cookbook-react/src/app/components/FormInput/index.tsx

```diff
@@ -14,12 +14,13 @@ interface Props {
   labelClassName?: string;
   name: string;
   onBlur?: (e: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
-  onChange: (e: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
+  onChange?: (e: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
   onFocus?: (e: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
   placeholder?: string;
   readOnly?: boolean;
   touched?: boolean;
   submitCount?: number;
+  inputRef?: any;
```

Comment:
Can you find a better type for this?

Reply:
Can you name it ref?

Reply:
Third try, if we want to maintain the props of an input you could use `Pick` like
```ts
interface Props extends Pick<InputProps, 'onBlur' | 'onFocus' > = {
   // Other props
}
```

Or use all the props of an input. What do you think?